ROCm_Tools/ROCm-Tools.rst

```diff
@@ -464,201 +464,244 @@ References
-   * Install c++filt using sudo apt-get install binutils
-
-ROCm Debugger Installation
-****************************
-1. If you did not install ROCm Debugger as part of the ROCm installation, you can download the ROCm Debugger debian packages 	 
-   (rocm-gpudebugsdk_<VERSION>_amd64.deb and rocm-gdb_<VERSION>_amd64.deb) independently and install them as follows.
-       * sudo dpkg -i rocm-gpudebugsdk_<VERSION>_amd64.deb
-       * sudo dpkg -i rocm-gdb_<VERSION>_amd64.deb
-           * The installed files will be placed in /opt/rocm/gpudebugsdk and /opt/rocm/gdb folders.
-       * Note that both rocm-gpudebugsdk and rocm-gdb debian packages are included as part of the ROCm repo install.
-2. Verify the setup
-
-   * Run the MatrixMultiplication sample provided in the GPU Debug SDK package
-         *  ``cd /opt/rocm/gpudebugsdk/samples/MatrixMultiplication``
-         *  ``make``
-         *  The Makefile assumes that the hsa header files are located at /opt/rocm/hsa/include. If you encounter a compilation failure, please 	    update the HSADIR within the Makefile to the directory of the hsa header files in the system.
-         *  Note that matrixMul_kernel.hsail is included for reference only. This sample will load the pre-built brig binary 		 	     (matrixMul_kernel.brig) to run the kernel.
-    * ``/opt/rocm/bin/rocm-gdb MatrixMul``
-         *  Tips: include the /opt/rocm/bin in your PATH environment variable
-
-Usage Examples
-################
-Check out the :ref:`tutorial` for some usage examples.
+The following environment variable can be used to enable dumping
+wavefront states when SIGINT (``ctrl c``) or SIGTERM (``kill -15``) is
+sent to the application:
 
-Known Issues
-###############
-   * Debugging hsa code objects that contain more than one BRIG module are not supported
-   * Debugging HSAIL kernels that contain global (or read only) variables are not supported
-   * Debugging HSAIL kernels that contain HSAIL function calls are not supported
-   * Using rocm-gdb objects in python scripts is not yet supported
-   * Single stepping branch instructions could require multiple step commands
+.. code:: sh
+
+   export ROCM_DEBUG_ENABLE_LINUX_SIGNALS=1
+
+Either of the following will disable this behavior:
+
+.. code:: sh
+
+   export ROCM_DEBUG_ENABLE_LINUX_SIGNALS=0
+   unset ROCM_DEBUG_ENABLE_LINUX_SIGNALS
+
+Code Object Saving
+------------------
+
+When the ROCr Debug Agent is enabled, each GPU code object loaded by the
+ROCm Platform Runtime will be saved in a file in the code object
+directory. By default the code object directory is
+``/tmp/ROCm_Tmp_PID_XXXX/`` where ``XXXX`` is the application process
+ID. The code object directory can be specified using the following
+environent variable:
+
+.. code:: sh
+
+   export ROCM_DEBUG_SAVE_CODE_OBJECT=code_object_directory
+
+This will use the path ``/code_object_directory``.
```

Comment:
Is this supposed to be "./code_object_directory" i.e. $CWD/code_object_directory?  Typically we just append /file_name to the user provided string to form a path to a file; we don't typically prepend a '/'.

Reply:
I'll just observe that prepending the '/' makes a value like "../dumps" pretty useless.

Reply:
Agreed:-)